packages/babel-parser/test/fixtures/typescript/interface/invalid-extends/input.ts

```diff
@@ -0,0 +1,2 @@
+// Every extend element here is invalid
+interface Invalid extends foo[bar], foo?.bar, foo(), import.meta {}
```

Comment:
Unfortunately the last example is currently accepted by Babel 7. In this PR we fix that for Babel 8.